azure-pipelines.yml

```diff
@@ -178,7 +178,8 @@ jobs:
     - template: _build/install-rust.yml
       parameters:
         platform: Linux
-        rust_version: nightly-2018-11-27
+        # least nightly release that stabilize try_from
+        rust_version: nightly-2019-02-26
```

Comment:
Updated nightly version to pass test. Is this acceptable? To say more, what's the policy of supported nightly version?

Reply:
Yeah, that's fine. This is only the compile tests. We don't officially support nightly, we just bump this periodically as needed to be able to run the compile tests (which require nightly)

Reply:
I see.